cmd/genpartial/main.go

```diff
@@ -50,9 +50,8 @@ func main() {
 	}
 
 	pkgs := Must(packages.Load(&packages.Config{
-		Dir:        cwd,
-		Mode:       mode,
-		BuildFlags: []string{"-tags=generate"},
```

Comment:
IIRC not excluding generated files really mucks with the build, especially if the generator produces invalid output. Or maybe it was that the original version would accidentally generate partial versions of the partials 🤔